config/routes.rb

```diff
@@ -27,4 +27,5 @@
 
   patch '/favorites/:pet_id', to: 'favorites#update'
   get '/favorites', to: 'favorites#index'
+  delete '/favorites/:pet_id', to: 'favorites#delete'
```

Comment:
I had to call this action delete because I had looked at your PR and saw that you had already used destroy, not sure what the right call is here as far as convention goes.

Reply:
I had exactly that same question. I was thinking we could have the following routes:

```
delete '/favorites', to: 'favorites#delete'
delete '/favorites/:pet_id', to: 'favorites#delete'
```

Reply:
In the favorites controller's delete action we could use the pet_id like so:

```
if params[:pet_id]
  session[:favorites].delete(params[:pet_id])
else
  session[:favorites] = nil
end
```

Reply:
But this is just me spitballing at midnight so I would love to hear from you on this.

Reply:
Oh I thought the action you made had to clear out all the favorites so it seemed to me that it would have to be a different action to delete just the one?

Reply:
I am not sure. I need to ask somebody about this.